app/prefs_window/prefs_window_layout.rb

```diff
@@ -0,0 +1,89 @@
+class PrefsWindowLayout < MK::WindowLayout
+  PREFS_WINDOW_IDENTIFIER = 'PREFSWINDOW'
+
+  def layout
+      frame from_center(size:[380, 220])
+      title "Whazzup Preferences"
+      style_mask (style_mask & ~NSResizableWindowMask)
+      add NSButton, :button_take_picture
+      add NSTextField, :button_take_picture_label
+      add NSTextField, :time_interval_label
+      add NSTextField, :time_interval
+  end
+
+  def time_interval_label_style
+    constraints do
+      width 80
+      height 20
+      left.equals(:superview, :left).plus(20)
+      top.equals(:superview, :top).plus(20)
+    end
+
+    editable false
+    selectable false
+    bordered false
+    bezeled false
+
+    cell do
+      alignment NSRightTextAlignment
+      scrollable false
+      drawsBackground false
+    end
```

Comment:
You should be able to extract lines 22-31 to a 'label_style' function, and just call it from each of the labels (line 74 below).